homework.js

```diff
@@ -0,0 +1,112 @@
+//--------- 1
+const summation = function(num){
+    let result = 0;
+    for (let i = 0; i <=num ; i++){
+        result += i;
+    }
+    return result;
+}
+//----------- 2
+const evenSummation = function(num){
+    let result = 0;
+    for (let i = 0; i <=num ; i+=2){
+        result += i;
+    }
+    return result;
+}
+evenSummation(5);
+
+//------------- 3
+
+let sum = 0;
+const list = [];
+const getAvg = function(list){
+    for (let i = 0; i < list.length; i++){
+        sum += list[i];
+    }
+    let avg = sum / list.length - 1;
+    return Math.floor(avg);

```

Comment:
no need for the `Math.floor()` as the average can be a float. just return the variable `avg`.